x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/fields/edit_field/edit_field.tsx

```diff
@@ -20,6 +20,7 @@ import {
   EuiCallOut,
 } from '@elastic/eui';
 
+import { documentationService } from '../../../../../../services/documentation';
```

Comment:
I understand why you made this change, but I would prefer for now to not couple the mappings editor with index management as, maybe the folder will be moved outside the app and in our `es_ui_shared` folder to make it available to other apps (like ML) for example.

Do you mind to revert the documentation methods to "lib" for now?

Reply:
I understand your concern. I worry leaving it as is though. We shouldn't be directly importing any `ui/xx` services anymore, unless there is a legacy API that isn't yet available in the NP, and in that case, it should be declared in the `shim`.

Reply:
I see, thanks for explaining. Let's leave it for now as you've done it then. I am going to leave a comment in the code for the `getGeoShapeDocumentationLink` method. Thanks 👍